packages/devextreme/testing/tests/DevExpress.ui.widgets/gallery.tests.js

```diff
@@ -385,15 +385,15 @@ QUnit.module('behavior', {
         assert.equal($loopItems.length, 2);
     });
 
-    QUnit.test('duplicate items should have role="presentation"', function(assert) {
+    QUnit.test('duplicate items should be hidden for readres', function(assert) {
         const $gallery = this.$element.dxGallery({
             items: [0, 1, 2, 3],
             loop: true
         });
 
         const $loopItems = $gallery.find(`.${GALLERY_LOOP_ITEM_CLASS}`);
         $loopItems.each((index, item) => {
-            assert.strictEqual($(item).attr('role'), 'presentation');
+            assert.strictEqual($(item).attr('aria-hidden'), 'true');
         });
     });
 
```

Comment:
[new, test coverage]
Let's add a testcafe test with Axe for base mods

Reply:
fixed